docker/addons/prometheus/grafana/example-dashboard.json

```diff
@@ -0,0 +1,451 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
```

Comment:
Add this for different services

![image](https://user-images.githubusercontent.com/28790446/231414689-74cda5f9-6fbb-4d90-97ec-c5707cda4bcd.png)